po/fr.po

```diff
@@ -0,0 +1,706 @@
+msgstr "erreur : option '{}' inconnue dans la section [bin]"
+
+#: src/config.rs:891
+msgid "error: unknown option '{}' in section [options]"
+msgstr "erreur : option '{}' inconnue dans la section [options]"
+
+#: src/config.rs:896
+msgid "option '{}' does not take a value"
+msgstr "l'option '{}' n'accepte pas de valeur"
+
+#: src/config.rs:947 src/install.rs:1796
+msgid "There are {n} providers available for {pkg}:"
+msgstr "Il y a {n} fournisseurs disponibles pour {pkg} :"
+
+#: src/config.rs:961 src/info.rs:57 src/info.rs:96 src/install.rs:1755
+#: src/install.rs:1804
+msgid "Repository"
+msgstr "Dépôt"
+
+#: src/config.rs:983
+msgid "  failed to sync {}"
+msgstr "  erreur de synchronisation de {}"
+
+#: src/devel.rs:129 src/download.rs:179
+msgid "Querying AUR..."
+msgstr "Interrogation de l'AUR..."
+
+#: src/devel.rs:142 src/devel.rs:166 src/install.rs:694
+msgid "failed to parse srcinfo for '{}'"
+msgstr "échec de l'analyse du srcinfo pour '{}'"
+
+#: src/devel.rs:184
+msgid "Looking for devel repos..."
+msgstr "Recherche des dépôts devel..."
+
+#: src/devel.rs:193
+msgid "failed to save devel info"
+msgstr "échec de l'enregistrement des informations devel"
+
+#: src/devel.rs:382
+msgid "package does not have an update"
+msgstr "le paquet n'a pas de mise à jour"
+
+#: src/devel.rs:417
+msgid "failed to lookup: {}"
+msgstr "erreur lors de la recherche : {}"
+
+#: src/devel.rs:445
+msgid "invalid json: {}"
+msgstr "json invalide : {}"
+
+#: src/download.rs:83
+msgid "packages not in the AUR: "
+msgstr "paquets introuvables dans l'AUR : "
+
+#: src/download.rs:94
+msgid "marked out of date: "
+msgstr "marqués comme pas à jour : "
+
+#: src/download.rs:105
+msgid "orphans: "
+msgstr "orphelins : "
+
+#: src/download.rs:206
+msgid "could not get current directory"
+msgstr "impossible de récupérer le dossier actuel"
+
+#: src/download.rs:210
+msgid "can not get repo packages: asp is not installed"
+msgstr "impossible de récupérer les paquets du dépôt : asp n'est pas installé"
+
+#: src/download.rs:231
+msgid "Missing ABS packages "
+msgstr "Paquets ABS manquants "
+
+#: src/download.rs:250 src/download.rs:276 src/download.rs:530
+#: src/download.rs:543 src/download.rs:557 src/exec.rs:72 src/exec.rs:113
+#: src/exec.rs:122 src/exec.rs:148 src/exec.rs:161 src/exec.rs:186
+#: src/exec.rs:200 src/exec.rs:221 src/exec.rs:234 src/install.rs:744
+#: src/install.rs:807 src/install.rs:845 src/keys.rs:74
+msgid "failed to run:"
+msgstr "échec du lancement :"
+
+#: src/download.rs:263
+msgid "does not contain PKGBUILD: skipping"
+msgstr "ne contient pas de PKGBUILD : ignorer"
+
+#: src/download.rs:294
+msgid " ({total>padding$}/{total}) downloading: {pkg}"
+msgstr " ({total>padding$}/{total}) téléchargement : {pkg}"
+
+#: src/download.rs:317
+msgid "Downloading PKGBUILDs..."
+msgstr "Téléchargement des PKGBUILDs..."
+
+#: src/download.rs:321
+msgid " PKGBUILDs up to date"
+msgstr " Les PKGBUILDs sont à jour"
```

Comment:
'PKGBUILDs à jour'

Wouldn't that be closer a translation ?

Reply:
Yes i will change that